django/forms/models.py

```diff
@@ -573,8 +573,9 @@ class BaseModelFormSet(BaseFormSet):
     unique_fields = set()
 
     def __init__(self, data=None, files=None, auto_id='id_%s', prefix=None,
-                 queryset=None, *, initial=None, **kwargs):
+                 queryset=None, *, edit_only=False, initial=None, **kwargs):
```

Comment:
Do you think it should be possible to pass in `edit_only` to the various formset factories?

Reply:
Do you mean pass in `edit_only` to (for example) two different model formsets or to formset and model formset?

Reply:
@smithdc1?